src/Symfony/Component/DependencyInjection/Dumper/XmlDumper.php

```diff
@@ -138,7 +138,7 @@ private function addService(Definition $definition, ?string $id, \DOMElement $pa
                 $tag = $this->document->createElement('tag');
                 $tag->setAttribute('name', $name);
                 foreach ($attributes as $key => $value) {
-                    $tag->setAttribute($key, $value);
+                    $tag->setAttribute($key, (string) $value);
```

Comment:
Let's keep this change as small as possible.

```suggestion
                    $tag->setAttribute($key, $value ?? '');
```

Reply:
Fine by me, I choose for casting because that is what's done in the rest of the file.